src/EpicsClient/EpicsClientMonitor.h

```diff
@@ -51,6 +51,9 @@ class EpicsClientMonitor : public EpicsClientInterface {
       ChannelInfo &channelInfo,
       std::shared_ptr<Ring<std::unique_ptr<FlatBufs::EpicsPVUpdate>>> ring);
   ~EpicsClientMonitor() override;
+  ///\fn emit
+  ///\param up an epics PV update struct holding the pv structure
+  ///\brief pushes the PV update onto the emit_queue ring buffer
```

Comment:
\brief should be first then the params.
Leave a line between the \brief and the params (as noted above you don't need the \brief.
The \param description should not be too coupled to the type and should be sentence form (capitial and fullstop), eg.
\param up The PV update to add to the queue.

Reply:
Technically the param description doesn't have to be sentence form but this usually results in some people doing it and others not, so the code base looks inconsistent.

I think it is better to pick a style and then enforce it.